google/cloud/spanner_v1/snapshot.py

```diff
@@ -20,7 +20,7 @@
 from google.cloud.spanner_v1.proto.transaction_pb2 import TransactionOptions
 from google.cloud.spanner_v1.proto.transaction_pb2 import TransactionSelector
 
-from google.api_core.exceptions import ServiceUnavailable
+from google.api_core.exceptions import ServiceUnavailable, InternalServerError
```

Comment:
We normally do one import per line, e.g.:
```suggestion
from google.api_core.exceptions import InternalServerError
from google.api_core.exceptions import ServiceUnavailable
```